generator-test/src/test/pegasus/com/linkedin/pegasus/generator/test/idl/enums/EscapedSymbols.pdl

```diff
@@ -5,7 +5,6 @@ namespace com.linkedin.pegasus.generator.test.idl.enums
  */
 enum EscapedSymbols {
   ENUM,
-  `enum`,
```

Comment:
why do we delete this one?

Reply:
This was an issue introduced in previous commit. enum cannot be used as it is reserved keyword in java. The template generation step fails for this.

Reply:
Oops